README.md

```diff
@@ -4,16 +4,64 @@ Model Assessment and Decision Analysis for Marine Environments web application.
 Prototype Decision Support app for the [ADRIA.jl](https://github.com/open-AIMS/ADRIA.jl)
 platform and CoralBlox model. This project also contains the Reef Guide app.
 
+## Table of Contents
+- [Quick Start](#quick-start)
+- [Angular Developer Setup](#angular-developer-setup)
+- [Development Server](#development-server)
+- [ReefGuide Web API Integration](#reefguide-web-api-integration)
+- [Code Scaffolding](#code-scaffolding)
+- [Configuration](#configuration)
+- [Build](#build)
+- [Deploy](#deploy)
+- [Testing](#testing)
+- [Further Help](#further-help)
+
+## Quick Start
+
+```bash
+# Clone the repository
+git clone https://github.com/open-AIMS/MADAME-app.git
```

Comment:
I'd assume people know how to do this. Also, you could do this using various methods such as VSCode git clone action.